tests/cases/fourslash/findAllRefsDestructureGetter2.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="fourslash.ts" />
+
+// @noLib: true
+
+// @Filename: /a.ts
+////class C {
+////    get [|{| "isWriteAccess": true, "isDefinition": true |}m|](): number { return 0; }
+////}
+////const { [|{| "isWriteAccess": true, "isDefinition": true |}m|] } = new C();
+
```

Comment:
Can you add another property with set accessor only ?

Reply:
:+1: